docs/en_us/internal/testing.rst

```diff
@@ -70,6 +70,12 @@ UI Acceptance Tests
    write end-user acceptance tests directly in Python, using the
    framework to maximize reliability and maintainability.
 
+Internationalization Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Any new text that is added should be internationalized and translated.
```

Comment:
I'm wondering if this should be in the [edx-documentation repo](https://github.com/edx/edx-documentation) instead? I'm not sure what the best place is for this in the repo, but I think it makes more sense than leaving it in edx-platform.